classes/log/td.php

```diff
@@ -67,7 +66,7 @@ public static function write($level, $msg, $method = null)
 	
 						//
 						if($level === 'Error'){
-							//if ($level == 'Error') var_dump($backtrace);
+							$msg = print_r($backtrace,true);
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* No space found after comma in function call